blocks/header/header.js

```diff
@@ -4,12 +4,8 @@ import { loadCSS, loadScript } from '../../scripts/aem.js';
  * loads and decorates the header, mainly the nav
  * @param {Element} block The header block element
  */
-export default async function decorate(block) {
-  // Global nav
-  const json = document.createElement('script');
-  json.appendChild(document.createTextNode('window.gnav_jsonPath = 20220-nav-config.25.json'));
-  block.appendChild(json);
+export default async function decorate() {
+  window.gnav_jsonPath = '2022-nav-config.25.json';
```

Comment:
Just realized, we need the full path: `  window.gnav_jsonPath = "https://main--esri--aemsites.hlx.live/2022-nav-config.25.json"`